AstSemantics.md

```diff
@@ -248,6 +295,32 @@ The details of index space for local variables and their types will be further c
 e.g. whether locals with type `i32` and `i64` must be contiguous and separate from
 others, etc.
 
+## Global variables
+
+A *global variable* stores a single value of a fixed [value type](#types) and may be
+declared either *mutable* or *immutable*. This provides WebAssembly with memory
+locations that are disjoint from any [linear memory](#linear-memory) and thus
+cannot be arbitrarily aliased as bits.
+
```

Comment:
Since no use case for mutable global variables has been made I do not support them being added back here. This matter had already been considered, and they were removed. Implementations are welcome to support them internally for asm.js but this need not be part of wasm. No new performance number have been presented after the decision to remove the global variables that suggests they were needed?